packages/hub/src/utils/WebBlob.ts

```diff
@@ -21,7 +21,7 @@ export class WebBlob extends Blob {
 		const customFetch = opts?.fetch ?? fetch;
 		const response = await customFetch(url, { method: "HEAD" });
 
-		const size = Number(response.headers.get("content-length"));
+		const size = Number(response.headers.get("x-linked-size") ?? response.headers.get("content-length"));
```

Comment:
revert?

Reply:
yes indeed :grimacing: done in https://github.com/huggingface/huggingface.js/pull/1230/commits/922bfdffb2e3f94ca2dc3fca93f2a0a5098ccee9